data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,1660 @@
+		require "Pug Peacekeeping Staff" 0
+		conversation
+			`As you land on this planet, you think back of the peculiar request He-Yi made to you: bring back non-deadly weapons. After thinking a while about it, you remember tales about the Pug having special weapons that could easily knock people out. Maybe you should go to <destination> and speak to the Oathkeepers about this.`
+				accept
+	to fail
+		has "Unfettered: Close Quarter: done"
+	on complete
+		conversation
+			`You cannot find Admiral Danforth, and wager he's probably busy with the pirates above. You think of disturbing him, but another Oathkeeper sees you and asks what you're looking for. You start speaking with him of the Pug war, and recognizing you, he seems eager to share his experience.`
+			`	"I was a simple soldier fighting with the rest of the fleet, in the Republic, at that time. We managed to find an isolated and disabled Pug ship. That by itself seemed like an opportunity too good to let go. Given we were in a Cruiser, we did not fear the crew of such a small ship. But we were wrong: as soon as we boarded said ship, we knew something was wrong. But we were committed, so we went in anyway. Being a soldier, I was on the frontline, and I saw things... Things you do not want to see, trust me. I saw people falling for no apparent reason. But that was not the worst of it. As you can see, I have survived, and that by itself is terrifying. They shot me! They shot me, and yet I lived."`
+			`	You thank him for telling you this story and then think to yourself. This means there is a good chance the Pug have the non-lethal weapons you are looking for. Now you just have to find the Pug, but they'll probably not give you these weapons willingly...`
+
+
+
+mission "Unfettered: Wanderer Homeworld"
+	invisible
+	source "Firelode"
+	destination "Vara K'chrai"
+	to offer
+		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
+	to fail
+		has "Unfettered: Speak Hai: failed"
+	waypoint "Ka'ch'chrai"
+	on waypoint
+		dialog `Judging by the population compared to the other worlds, this appears to be the capital of these aliens. You should land to introduce yourself as an diplomat between them and the Hai.`
+
+mission "Unfettered: Speak Hai"
+	name `Meet the Wanderers and acquire a translator`
+	description `You need to be able to act as a diplomat between the Hai and the aliens up north. First go and meet those aliens. Then go to <stopovers> to see if He-Yi will provide a translator for you.`
+	source "Firelode"
+	stopover "Warfeed"
+	to offer
+		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
+		not "language: Wanderer"
+	on offer
+		conversation
+			`Moving through the spaceport, you notice some buildings are not as dilapidated as before. You wonder if the Unfettered are the ones who invested into repairs, or if the new Hai arrivals brought this about. Whatever the case is, this does not look like a place of conflict anymore. It looks like a place of hope.`
```

Comment:
You really love `dilapidated`, don't you? It might still be better to change it to something else in some of the descriptions.

Reply:
Its not me its a reviewer
Yeah idk I dont know synonyms or anything

Reply:
I used synonyms from online